components/ArticlesView.tsx

```diff
@@ -68,6 +69,7 @@ const ArticlesView: React.ElementType<ArticlesViewProps> = ({
   displayLoadMore = true,
   displayExcerpt = true,
   displayDateAuthor = true,
+  hideCategory = false,
```

Comment:
I don't think we should add `hideCategory` here -- can't we just use `displayCategory` which is already a boolean value?